cypress/integration/tests/login.test.ts

```diff
@@ -3,7 +3,7 @@
 import { login } from "../../utils/utils";
 
 describe("Log In", () => {
-    it("Login to Pathfinder", () => {
+    it("Login to Pathfinder", { tags: "@tier1" }, () => {
```

Comment:
I skipped this as it will definitely get verified (and run) irrespective of the tier along with all the specs.

Reply:
@nitishSr I agree. but
I added it because it creates confusion when you calculates tests.
It shows 131 tests but 130 executed.
Do you still think - I should remove it from tiers?
Please let me know.

Reply:
That's fine, it hardly takes few seconds to run